### PR TITLE
*: support allowed Common Name(CN)

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
 support allowed Common Name(CN)

### What is changed and how it works?
add filed `allowed CN`

### Check List 

Tests 
 - Manual test

### CODE REVIEW

File: pkg/grpcutil/grpcutil.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ type SecurityConfig struct {
 	// KeyPath is the path of file that contains X509 key in PEM format.
 	KeyPath        string `toml:"key-path" json:"key-path"`
 	ClientCertAuth bool   `toml:"client-cert-auth" json:"client-cert-auth"`
+	AllowedCN      string `toml:"allowed-cn" json:"allowed-cn"`
```
Comment (weekface) [inline pkg/grpcutil/grpcutil.go]: Can you change `allowed-cn` to `cert-allowed-cn` like other components?

Comment (nolouch) [reply pkg/grpcutil/grpcutil.go]: ptal

**Pull request merged**